### DISCUSSION TITLE
Search broken?

### DISCUSSION THREAD

**Discussion by talonx** [daattali/beautiful-jekyll#1230] [Q&A]
My site repo is https://github.com/talonx/talonx.github.io/ and the site is live at https://code.deepinspace.net/

The search function does not seem to work at all. The search box comes up but nothing more happens when I type or press Enter. Anything I might be doing wrong?

Firefox 118.0.2 on Ubuntu 22 (snap).

**Comment by systemcenterblog** [comment]
I have just setup my site too and have the same issue.

**Comment by daattali** [comment]
Thanks for reporting. I can see from a quick look at the javascript console on your webpage that this is likely an issue with not properly escaping the search information. 

This is what I'm seeing in the console:

![image](https://github.com/daattali/beautiful-jekyll/assets/952340/fae3ad33-ab1a-4d08-94fe-8a529f1c4b71)

And looking at the HTML, I see this:

![image](https://github.com/daattali/beautiful-jekyll/assets/952340/6413df28-37f6-456f-a383-28ed4f88a0e0)

I'll try to take a look to fix when I find time. Feel free to help with a PR as well if you can.

**Comment by daattali** [comment]
This has been fixed. Please confirm.

**Reply by systemcenterblog** [reply]
https://github.com/systemcenterblog/systemcenterblog.github.io

I experience the same issue however, i do not have Escaped ' char issue.  I have applied the new Search.html without success.

**Reply by systemcenterblog** [reply]
I have removed all my HTML posts that were converted from Blogger so only .MD posts are present and the search feature is working.  What Character break the search script?  i.e. _ ' ._ [] $
